src/misc/sphere-collider.js

```diff
@@ -50,6 +50,7 @@ module.exports = {
       if (!mesh) { return; }
 
       position.copy(el.object3D.getWorldPosition());
+      mesh.getWorldPosition(position);
```

Comment:
This shouldn't be needed - another PR a couple days (adding the `el.object3D.getWorldPosition()` part) should have fixed this.